packages/suite/src/actions/wallet/sendFormActions.ts

```diff
@@ -96,6 +99,70 @@ export const removeDraft = () => (dispatch: Dispatch, getState: GetState) => {
     }
 };
 
+// TODO: replace by structuredClone() after updating TS
+const clone = <T>(info: T): T => {
```

Comment:
this one seems to belong to utils package to be used everywhere (there are lot of places where `JSON.parse(JSON.stringify(object))` is used, [i.e. 20 lines above](https://github.com/trezor/trezor-suite/pull/5970/files#diff-38581ddf07235dae32f96fa731c91cd468abdde80382d5a0571616d64662ef53R85) ), but since it makes sense to replace it soon by `structuredClone`, let's leave it like this...